pages/api/submissions/index.ts

```diff
@@ -0,0 +1,30 @@
+import forms from 'data/flexibleForms/forms';
+import { NextApiRequest, NextApiResponse } from 'next';
+import StatusCodes from 'http-status-codes';
+
+const handler = async (
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> => {
+  switch (req.method) {
+    case 'POST':
+      // TODO: start a new submission here
+      // TODO: send back new case object
+      res.json({
+        id: 1,
+      });
+      break;
+    case 'GET':
+      res.json({
+        forms,
```

Comment:
Ah, I see what you have in your PR notes which answers this q:
>new internal api endpoint which will eventually handle starting a fresh submission and providing a list of the possible forms to power the page

I still think `/api/submissions` is a bad name (because it doesn't return submissions). Maybe we could think of something more descriptive for the actual data you get back?

Reply:
no, it will return submissions eventually - and probably people to boot. these apis don't just return one domain/model. they'll return the right mix of things that the page needs to render everything quickly.

Reply:
plus, mirroring the routing structure of the frontend is super helpful when debugging. you can just drop the `/api` prefix in and inspect exactly what data the page is getting

Reply:
you're only seeing an incomplete, stunted version of these endpoints at the moment. they'll look like this when done: https://github.com/LBHackney-IT/social-care-recording/tree/main/pages/api/submissions